classroom-dashboard/app/views/sections/_form.html.erb

```diff
@@ -2,7 +2,7 @@
 <br>
 <br>
 <br>
-<!-- fix this to better form lol  -->
+<!-- fix this to better form once we meet with client -->
```

Comment:
Good update, only thing I'd suggest is appending something uniform to these replaceable comments, such as "TODO" or something so that it can be later verified that they were removed.